Updated to RAC 3 beta 6

alskipp opened the pull request.

alskipp committed:
Updated to RAC 3 beta 6

alskipp committed:
QueueScheduler requires argument label

alskipp committed:
Removed 'textToSearchSignal' let declaration

Explicit type declaration is no longer required.

alskipp commented:
Upon updating to the latest RAC I had a build error relating to `textToSearchSignal`. Changing from `SignalProducer` to a plain `Signal` fixed the error.

This seems reasonable to me, though I'm not very familiar with RAC yet, so please let me know if you think a different approach would be better.

ColinEberhardt commented:
Thanks for the updates, I'll take a look tonight. Maybe the beta 6 changes mean the flatMap can be placed in the pipeline once again? I only split it out because type inference was unable to infer the correct flatMap implementation.

alskipp commented:
I see, in that case it probably makes sense to lose the 'let' and integrate it into the full expression. 
(I'm out and about at the minute, but will try that amend, if I get back early enough).

alskipp committed:
Updated .gitignore

Ignore *.xccheckout to prevent merge conflicts.

alskipp committed:
Removed local path from FRAMEWORK_SEARCH_PATHS

Removed the following path:
/Users/colineberhardt/Projects/ReactiveSwiftSummit/Carthage/Build/iOS

Carthage frameworks are linked correctly already with:
$(PROJECT_DIR)/Carthage/Build/iOS
